lib/matplotlib/_constrained_layout.py

```diff
@@ -602,8 +604,8 @@ def get_pos_and_bbox(ax, renderer):
 
     Parameters
     ----------
-    ax
-    renderer
+    ax : Axes
+    renderer : Renderer
```

Comment:
should these be `~matplotlib.axes.Axes` and `~matplotlib.backend_bases.RendererBase`?

Reply:
Sure, but they are in a private module and I have another PR stalled for adding the fully qualified paths, so I selected to take, what appeared to be, the easy way.

(Is this PR an improvement compared to the previous?)

Reply:
Now that it is established that we do not merge PRs without fully qualified paths (even in private modules), can please #26334 be considered again (which has been stalled for almost two months because it used that)?

(Yes, it needs to be rebased, but no point in doing that if it will just be stalled further.)